apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -354,7 +354,6 @@ static AppConfiguration readConfigFromEnvironmentVariables() {
     ExtractionOptions extractionOptions = new ExtractionOptions(s3BucketName, allowedRifFileType);
```

Comment:
Need to remove `.toCharArray()` on the line above this one now since the `DatabaseOptions` change.

````
new DatabaseOptions(databaseUrl, databaseUsername, databasePassword.toCharArray());
````